### ISSUE TITLE
[Bug]:  module 'llama_index.core' has no attribute '__version__' - in non legacy ElasticsearchStore

### ISSUE THREAD

**Issue by Jotschi** [run-llama/llama_index#10639]
### Bug Description

The elasticsearch implementation *still* tries to access llama_index.core.__version__.

https://github.com/run-llama/llama_index/blob/v0.10.1/llama-index-integrations/vector_stores/llama-index-vector-stores-elasticsearch/llama_index/vector_stores/elasticsearch/base.py#L233

In 0.9.48 this has been set in the `__init__.py`

https://github.com/run-llama/llama_index/blob/v0.9.48/llama_index/__init__.py#L5

### Workaround

```python
import llama_index.core as lcore
lcore.__version__ = "0.10.1"
```

### Version

0.10.1

### Steps to Reproduce

Instantiate `ElasticsearchStore`.

### Relevant Logs/Tracbacks

```shell
AttributeError: module 'llama_index.core' has no attribute '__version__'
Traceback:
File "/home/jotschi/workspaces/rag/rag/llama-index-streamlit/venv/lib/python3.11/site-packages/streamlit/runtime/scriptrunner/script_runner.py", line 535, in _run_script
    exec(code, module.__dict__)
File "/home/jotschi/workspaces/rag/rag/llama-index-streamlit/src/streamlit-rag.py", line 318, in <module>
    main()
File "/home/jotschi/workspaces/rag/rag/llama-index-streamlit/src/streamlit-rag.py", line 309, in main
    vs = setupVectorStore(vector_store_type, vector_store_url, vector_store_index_name)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/jotschi/workspaces/rag/rag/llama-index-streamlit/src/rag.py", line 50, in setupVectorStore
    return ElasticsearchStore(vector_store_index_name, es_url=vector_store_url)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/jotschi/workspaces/rag/rag/llama-index-streamlit/venv/lib/python3.11/site-packages/llama_index/vector_stores/elasticsearch/base.py", line 197, in __init__
    self._client = _get_elasticsearch_client(
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/jotschi/workspaces/rag/rag/llama-index-streamlit/venv/lib/python3.11/site-packages/llama_index/vector_stores/elasticsearch/base.py", line 84, in _get_elasticsearch_client
    **connection_params, headers={"user-agent": ElasticsearchStore.get_user_agent()}
                                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "/home/jotschi/workspaces/rag/rag/llama-index-streamlit/venv/lib/python3.11/site-packages/llama_index/vector_stores/elasticsearch/base.py", line 233, in get_user_agent
    return f"llama_index-py-vs/{llama_index.core.__version__}"
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
```

**Comment by logan-markewich** [comment]
@Jotschi thanks, will fix this shortly

**Comment by logan-markewich** [comment]
@Jotschi this is fixed, please install v0.1.2 of the vector store -- `pip install -U llama-index-vector-stores-elasticsearch`
